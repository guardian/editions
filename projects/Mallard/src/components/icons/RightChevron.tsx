import React from 'react';
import { StyleSheet, Text } from 'react-native';
import { useAppAppearance } from 'src/theme/appearance';
import { getFont } from 'src/theme/typography';

const RightChevron = () => {
	const styles = StyleSheet.create({
		icon: {
			...getFont('icon', 1),
			color: useAppAppearance().color,
		},
	});

	return <Text style={styles.icon}>{'\uE00B'}</Text>;
};

export { RightChevron };
