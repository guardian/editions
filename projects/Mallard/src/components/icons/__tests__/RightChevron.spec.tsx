import React from 'react';
import TestRenderer from 'react-test-renderer';
import { RightChevron } from '../RightChevron';

describe('RightChevron', () => {
	it('should display a RightChevron icon using the icon font', () => {
		const component = TestRenderer.create(<RightChevron />).toJSON();
		expect(component).toMatchSnapshot();
	});
});
