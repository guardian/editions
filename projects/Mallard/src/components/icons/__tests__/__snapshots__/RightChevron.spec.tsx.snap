// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RightChevron should display a RightChevron icon using the icon font 1`] = `
<Text
  style={
    {
      "color": "#121212",
      "fontFamily": "GuardianIcons-Regular",
      "fontSize": 20,
      "lineHeight": 20,
    }
  }
>
  î€‹
</Text>
`;
